addons/sale/models/sale_order_line.py

```diff
@@ -652,6 +659,38 @@ def _compute_qty_delivered(self):
         for so_line in lines_by_analytic:
             so_line.qty_delivered = mapping.get(so_line.id or so_line._origin.id, 0.0)
 
+    def _downpayment_state(self):
+        """
+        Check that this sale order line is a draft down payment
+        Note: self.ensure_one()
+        :return: True Verify that this SO line is a draft down payment
+        :rtype: Boolean
+        """
```

Comment:
Unclear method name, maybe you meant `_get_downpayment_state`.
Wrong docstring, you're returning strings and not booleans